### ISSUE TITLE
Hardcoding of events interval

### ISSUE THREAD

**Issue by RRothfeld** [BauhausLuftfahrt/MATSim-UAM#63]
https://github.com/BauhausLuftfahrt/MATSim-UAM/blob/815269590a0d9fa37b782c332a6e19e0ef882780/src/main/java/net/bhl/matsim/uam/run/RunUAMScenario.java#L124

Is it necessary to hard-code the events interval?

**Comment by sebhoerl** [comment]
No, but also not all the other stuff. There should be a run script that only adds the modules but does not modify any config. This script here is only an example, in any case. We can build it back and ideally not modify any config here.
